lib/business_time/currency.rb

```diff
@@ -1,26 +1,27 @@
 module BusinessTime
   module Currency
+
     # @return [Array] the list of currencies
     # @param [String+]
     def args(*currency)
-      return [] if currency.empty?
+      return Set.new if currency.empty?
```

Comment:
Don’t be afraid of rubocop complaining about function name starting with capital letter. `Integer()` is _a function_, defined on `Kernel` module. Here is ruby, that’s how we get things done. Shut the rubocop up with a comment (or, well,  yes, change the name to start with lowercase :)